lib/components/narrative/narrative-routing-results.js

```diff
@@ -32,12 +31,8 @@ class NarrativeRoutingResults extends Component {
     if (mainPanelContent) return null
 
     return (
-      routingType === 'ITINERARY'
-        ? <TabbedItineraries itineraryClass={itineraryClass} itineraryFooter={itineraryFooter} itineraries={itineraries} customIcons={customIcons} />
-        : <NarrativeProfileOptions
-          itineraryClass={itineraryClass}
-          customIcons={customIcons}
-        />
+      routingType === 'ITINERARY' &&
+        <TabbedItineraries itineraryClass={itineraryClass} itineraryFooter={itineraryFooter} itineraries={itineraries} customIcons={customIcons} />
```

Comment:
Probably warrants putting this on multiple lines (seems to violate the characters per line limit).